Formula/picotool.rb

```diff
@@ -0,0 +1,54 @@
+class Picotool < Formula
+  desc "Tool for interacting with a x RP2040 device in BOOTSEL mode, or with a RP2040 binary"
```

Comment:
Sorry, the audit [didn't like this description](https://github.com/Homebrew/homebrew-core/actions/runs/4394510772/jobs/7695597730#step:4:16) because it was too long:

```suggestion
  desc "Tool for interacting with RP2040 devices in BOOTSEL mode, or RP2040 binaries"
```

Or, you can come up with one, if you have a better idea.